docs/lock-machines.mdx

```diff
@@ -2,7 +2,7 @@
 title: Lock machines
 ---
 
-You can lock a machine anytime to prevent accidental edits. Locked machines have a <Lock size={18} /> **Locked** status displayed in the editor’s top bar.
+You can lock a machine anytime to prevent accidental edits. Locked machines have a <Lock size={18} /> **Locked** status displayed in the editor’s top bar. They also have a lock icon in the project’s machine list in the left panel.
```

Comment:
```suggestion
You can lock a machine anytime to prevent accidental edits. Locked machines have a <Lock size={18} /> **Locked** status displayed in the editor’s top bar and in the project’s machine list in the left panel.
```